lib/vehicle.js

```diff
@@ -461,7 +461,7 @@ Vehicle.prototype.vin = function() {
 /**
  * POST Vehicle.lock
  *
- * @return {Promise.<Security>} response on successful request
+ * @return {Promise.<ActionSuccess>} response on successful request
```

Comment:
I think you also need to change the typdef above that is named `Security` still on line 452!

Reply:
sorry, unpushed changes..just pushed in.